<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background:rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="view_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1101</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color :rgb(255, 7, 65);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>303</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 2px solid rgb(93, 128, 255);
    border-radius: 10px;
    padding: 2px 6px;
    background-color: #FFFFFF;
    font-size: 20px;
    color: #000000;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 50px;
    border-left: 1px solid rgb(194, 226, 255);
    background-color: rgb(255, 255, 255);
	border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QComboBox::down-arrow {
    image: url(&quot;C:/plotting/Icon/arrow.png&quot;);
    width: 40px;
    height: 40px;
}

QComboBox QAbstractItemView {
    border: 1px solid #A0A0A0;
    background-color: #FFFFFF;
    selection-background-color: #CCE5FF;
    selection-color: #000000;
    font-size: 20px;
    outline: 0;
	border-radius: 10px;
    background-clip: padding-box;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cosine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tangent</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cotangent</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="enter_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>700</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter angle :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="angle_lineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>720</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border :2px solid rgb(221, 221, 221);
	border-radius : 10px;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
